multiversx_sdk_cli/transactions.py

```diff
@@ -55,6 +58,48 @@ def __init__(self) -> None:
 
 
 def do_prepare_transaction(args: Any) -> Transaction:
+    account = load_sender_account_from_args(args)
+    transfers = getattr(args, "token_transfers", None)
```

Comment:
Maybe work with empty lists instead of `None` (should work the same).

Reply:
replaced with empty list